src/trait_/eq.rs

```diff
@@ -13,4 +16,38 @@ impl TraitImpl for Eq {
 	fn default_derive_trait(&self) -> DeriveTrait {
 		DeriveTrait::Eq
 	}
+
+	fn supports_skip(&self) -> bool {
+		true
+	}
+
+	fn build_signature(
+		&self,
+		_item: &Item,
+		_traits: &[DeriveTrait],
+		_trait_: &DeriveTrait,
+		body: &TokenStream,
+	) -> TokenStream {
+		quote! {
+			#[inline]
+			fn assert_receiver_is_total_eq(&self) {
```

Comment:
do we need to add the doc hidden as well?

Reply:
Good question, I will check.

Reply:
It doesn't seem to be necessary, doesn't show up in docs at all.